src/styles/base/_typography.scss

```diff
@@ -108,6 +108,12 @@ li li {
   font-size: 1em;
 }
 
+// make sure that task list checkboxes don't push the rest of the content to the next line
+li input[type="checkbox"] + p {
```

Comment:
I think we should also remove the default bullet that comes from unordered list items in addition to adding the checkbox. Any objections?

Reply:
That's fine by me, though i'm unfamiliar with the CSS needed to accomplish that.